package.json

```diff
@@ -18,6 +19,8 @@
     "react-dom": "^16.8.0-0"
   },
   "dependencies": {
+    "@types/react": "^16.9.32",
```

Comment:
These need to be included as dependencies https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies